flytekit/image_spec/image_spec.py

```diff
@@ -74,13 +84,19 @@ class ImageSpec:
     entrypoint: Optional[List[str]] = None
     commands: Optional[List[str]] = None
     tag_format: Optional[str] = None
+    copy: Optional[CopyFileDetection] = None
```

Comment:
This [PR](https://github.com/flyteorg/flytekit/pull/2715/files) already uses `copy`. should we use the other name?

Reply:
can you suggest a good name?

Reply:
yeah copy is bad because the `--copy` switch on the command line has the opposite effect.  --copy none on the command line means it should copy files into the image.

Reply:
how about `source_copy_mode` and `addl_copies` for the other pr?

Reply:
These names sound good to me. 👍